test/functionalTest/cases/0000_statistics_operation/statistics_with_counters.test

```diff
@@ -249,7 +249,7 @@ orionCurl --url /v1/updateContext                -X POST --payload "$payload"  >
 
 # Get statistics
 
-orionCurl --url /statistics?options
+orionCurl --url /statistics
```

Comment:
Not directly related with the purpose of this PR, this is just a typo fix.

NTC